themes/navy/layout/partial/head.swig

```diff
@@ -32,7 +32,7 @@
   {{ css('css/navy') }}
   <!-- endbuild -->
   <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700" rel="stylesheet" type="text/css">
```

Comment:
not sure the performance of googleapis.com over there, maybe you can consider npm-ed fonts.

https://github.com/KyleAMathews/typefaces/
https://github.com/khanhtq/lato-font-face/
https://github.com/betsol/lato-font/
https://github.com/Dilatorily/lato-webfont/